airbyte-test-utils/src/main/java/io/airbyte/test/utils/GKEPostgresConfig.java

```diff
@@ -52,7 +52,7 @@ public static Database getSourceDatabase() {
 
   public static Database getDestinationDatabase() {
     return new Database(DSLContextFactory.create(USERNAME, PASSWORD, DatabaseDriver.POSTGRESQL.getDriverClassName(),
-        "jdbc:postgresql://localhost:3000/postgresdb", SQLDialect.POSTGRES));
+        "jdbc:postgresql://localhost:4000/postgresdb", SQLDialect.POSTGRES));
```

Comment:
what are we trying to solve by changing this number?

Reply:
Ah I just read https://github.com/airbytehq/airbyte-cloud/pull/1943#discussion_r1003631056.

Can we add this to the PR description when we merge this in so it's clear in commit history? Thanks!